ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/DataColumnSidecarByRootCustody.java

```diff
@@ -15,11 +15,13 @@
 
 import java.util.Optional;
 import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.async.stream.AsyncStream;
 import tech.pegasys.teku.spec.datastructures.blobs.versions.fulu.DataColumnSidecar;
+import tech.pegasys.teku.spec.datastructures.blocks.SlotAndBlockRoot;
 import tech.pegasys.teku.spec.datastructures.networking.libp2p.rpc.DataColumnIdentifier;
 import tech.pegasys.teku.spec.datastructures.util.DataColumnSlotAndIdentifier;
 
-public interface DataColumnSidecarByRootCustody extends DataColumnSidecarCustody {
+public interface DataColumnSidecarByRootCustody extends UpdatableDataColumnSidecarCustody {
```

Comment:
Maybe we should merge interfaces  UpdatableDataColumnSidecarCustody to DataColumnSidecarCustody just to make it simpler?